app/javascript/components/game/game.css

```diff
@@ -4,22 +4,22 @@ body {
   font-family: Helvetica;
   font-weight: bold;
   text-align:center;
+  background-color: rgb(17, 17, 20);
 }
 
-.game_left {
+/* .game_left {
     width: 50%;
     height: 100%;
-    background: #485b6e;
     display: flex;
     flex-direction: column;
     align-items: center;
     justify-content: center;
-  }
+  } */
   
-  .game_right {
+  /* .game_right {
     width: 50%;
     background-size: 100% 100%;
-  }
+  } */
```

Comment:
Feel free to delete any commented-out lines. You can always retrieve them from Git history.